assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -29,6 +29,7 @@ min_perf_pct=100
 
 {{if .RealTime}}
 [service]
+uname_regex=rt
```

Comment:
stalld is not RT kernel specific, it can be useful for any system with FIFO workloads

Reply:
I am getting an error when applied stalld enabled in a vm without rt kernel.